github/lib/configure_repo.rb

```diff
@@ -101,6 +101,16 @@ def required_status_checks
              .fetch("additional_contexts", [])
         ].compact
       }
+    elsif github_actions_exists?
+      {
+        strict: false, # "Require branches to be up to date before merging"
```

Comment:
Thanks for the comment 👍 not a very self-documenting API otherwise!

Reply:
Hopefully the overrides setting I changed it to (`up_to_date_branches`) also works to document what it does. :thinking:

Reply:
It does 😄